command_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package updater
+
+import (
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/keybase/go-logging"
+)
+
+var log = logging.Logger{Module: "test"}
+
+func TestEmptyRunCommand(t *testing.T) {
+	out, err := RunCommand("", nil, time.Second, log)
+	if out != "" {
+		t.Errorf("Unexpected output: %s", out)
```

Comment:
order matters with `DeepEqual` but not with `JSONEq`. I've had random test failures with `DeepEqual` based on order of key insertion.